File: APP/app/src/main/java/com/example/myloginapp/MainActivity.java
package com.example.myloginapp;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;
import android.widget.Toast;

import com.google.android.material.button.MaterialButton;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


        TextView username = findViewById(R.id.username);
        TextView password = findViewById(R.id.password);

        MaterialButton loginbtn = findViewById(R.id.loginbtn);

        //admin and admin

        loginbtn.setOnClickListener(v -> {
            if(username.getText().toString().equals("admin") && password.getText().toString().equals("admin")){
                //correct
                Toast.makeText(MainActivity.this,"LOGIN SUCCESSFUL",Toast.LENGTH_SHORT).show();
                Intent intent = new Intent(MainActivity.this, RedirectPage.class);
                startActivity(intent);
            }else
                //incorrect
                Toast.makeText(MainActivity.this,"LOGIN FAILED !!!",Toast.LENGTH_SHORT).show();
        });


    }
}

File: P4_Med_Hist/P4_Med_Hist/app/src/main/java/com/example/p4_med_hist/MainActivity.kt
package com.example.p4_med_hist

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import androidx.activity.ComponentActivity


class MainActivity : ComponentActivity() {
    private lateinit var n: EditText
    private lateinit var a: EditText
    private lateinit var e: EditText
    private lateinit var h: EditText
    private lateinit var w: EditText

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        n = findViewById(R.id.name);
        a = findViewById(R.id.age);
        e = findViewById(R.id.email);
        h = findViewById(R.id.height);
        w = findViewById(R.id.weight);

        val nextPageButton: Button = findViewById(R.id.next)

        nextPageButton.setOnClickListener {
            val intent = Intent(this, MainActivity3::class.java)
            intent.putExtra("n", n.toString())
            intent.putExtra("a", a.toString())
            intent.putExtra("e", e.toString())
            intent.putExtra("h", h.toString())
            intent.putExtra("w", w.toString())
            startActivity(intent)
        }
    }
}